File: Algorithm/Leetcode/0004.cpp
// 1431. Kids With the Greatest Number of Candies

class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int max = *max_element(std::begin(candies), std::end(candies)); 
        vector<bool> output;
            for(auto i :candies )
            {
                output.push_back(i + extraCandies >= max );
            }
        return output;
    }
};

File: Algorithm/Leetcode/0000_0100/0000_0010/0004.py
## 1431. Kids With the Greatest Number of Candies

"""https://leetcode.com/problems/kids-with-the-greatest-number-of-candies/discuss/792205/Python-Most-Optimum-Solution"""

class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        	high_enough = max(candies) - extraCandies
	        return [i >= high_enough for i in candies]